integration-tests/bats/remotes.bats

```diff
@@ -1351,7 +1349,7 @@ SQL
     [ "$status" -eq 0 ]    
 }
 
-@test "remotes: validate that a config isn't needed for a pull." {
+@test "remotes: validate that a config is needed for a pull." {
```

Comment:
Is it the case that config wasn't required before? This is surprising.

Reply:
It seems that config wasn't required to do a fast-forward pull before, but merge always requires a config so I think it makes sense for pull to do the same. It's also hard to allow fast-forward pulls without a config because that logic would have to be taken care of in the sql context.

Reply:
Agreed. Let's alway require it.